rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -895,6 +968,7 @@ void MasterGraph::output_routine()
 #endif
     }
     try {
+        _process_time.start();
```

Comment:
Remove this change. This is not part of this PR.

Reply:
Resolved in the latest commit